src/styles/components/application-preloader/styles.less

```diff
@@ -65,3 +66,17 @@
     width: 100px;
   }
 }
+
+& when (@screen-resolution-2x-enabled) {
+
+  @media only screen and (min-resolution: 2dppx) {
+
+    & when (@application-preloader-enabled) {
+
+      .application-preloader-spinner {
```

Comment:
We shouldn't include the 2x version as a base64 string as we're currently only generating one css file for all resolutions meaning we would always load both versions of the spinner even if only one is needed.

Reply:
removed

Reply:
with our current seupt it is not (easily) possible to parse the CSS for our bootstrap-page, this is the reason why referencing images will caus a 404. including the images as base64 isnt the best approach, but it works for now. JIRA: https://jira.mesosphere.com/browse/DCOS_OSS-1564